.ci/docker/build.sh

```diff
@@ -381,6 +381,13 @@ case "$image" in
     HALIDE=yes
     TRITON=yes
     ;;
+  pytorch-linux-jammy-py3.12-triton-cpu)
+    CUDA_VERSION=12.4
```

Comment:
You probably don't want CUDA here, do you?
```suggestion
```

Reply:
This seems to have caused an error due to libmpi_cxx not being found. I think it got installed as part of the CUDA PyTorch install. I guess I could figure out how to install it independently, but just installing CUDA seems easier